apps/navstar_l4lb/src/navstar_l4lb_app.erl

```diff
@@ -12,17 +12,17 @@
 
 start(_StartType, _StartArgs) ->
     load_config_files(),
-    maybe_start_minuteman().
+    maybe_start_l4lb().
 
 stop(_State) ->
     ok.
 
-maybe_start_minuteman() ->
-    case application:get_env(navstar, enable_lb, false) of
+maybe_start_l4lb() ->
+    case application:get_env(navstar, enable_lb, true) of
```

Comment:
looks like the dcos value is true. I want to change this logic... it should be true by default. If you don't want then set it to false